api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -59,22 +45,21 @@ public interface WorkspaceMapper {
   @Mapping(target = "cdrVersionId", source = "dbWorkspace.cdrVersion")
   @Mapping(target = "accessTierShortName", source = "dbWorkspace.cdrVersion.accessTier.shortName")
   @Mapping(target = "googleProject", source = "dbWorkspace.googleProject")
-  @Mapping(target = "published", ignore = true)
-  @Mapping(target = "featuredCategory", ignore = true)
+  @Mapping(target = "featuredCategory", ignore = true) // set by setFeaturedWorkspaceCategory()
   Workspace toApiWorkspace(
       DbWorkspace dbWorkspace,
       RawlsWorkspaceDetails fcWorkspace,
       FeaturedWorkspaceService featuredWorkspaceService);
 
   @AfterMapping
-  default void setFeaturedWorkspaceInfo(
+  default void setFeaturedWorkspaceCategory(
       @MappingTarget Workspace workspace,
       DbWorkspace dbWorkspace,
       FeaturedWorkspaceService featuredWorkspaceService) {
-    boolean isPublished = featuredWorkspaceService.isFeaturedWorkspace(dbWorkspace);
```

Comment:
semantic change: we no longer consider new-style featured workspaces to be "published" because that term is reserved for old-style